### ISSUE TITLE
Weird tile numbering

### ISSUE THREAD

**Issue by Jocke-G** [perliedman/ocad2tiles#1]
I was trying to put my genereted map tiles on top of open street map. Started with a blank angular project, and [ngx-leaflet](https://github.com/Asymmetrik/ngx-leaflet). Added a layer with my map tiles. But when leaflet tries to reach my tiles, it requests totally different numbers.

**Comment by perliedman** [comment]
Yeah, the issue most likely is with the coordinate reference system (CRS). For now, `ocad2tiles` will use the CRS of the OCAD file, which is different from the CRS Leaflet uses (Spherical Mercator or EPSG:3857).

To get tiles in the same CRS as Leaflet (and OpenStreetMap's default tiles), we would have to reproject the OCAD data to spherical mercator and then draw the tiles. This is absolutely possible, and I've done it before in other projects, but there are some downsides as the OCAD maps are usually drawn with a particular CRS in mind; text gets rotated slightly and so on.

Having said that, I think having the option to output tiles in spherical mercator would be a great addition. Can't promise I'll add it myself at the moment since I'm quite busy with other stuff, but I'd be happy to give some pointers if you are interested.
